Banking/src/main/java/com/example/Banking/model/Account.java

```diff
@@ -0,0 +1,130 @@
+package com.example.Banking.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Table(name = "accountdetails2")
+public class Account {
+
+    private long id;
```

Comment:
I would recommend annotate fields with column names and avoid annotating getters. And instead of adding getters, setters and toString by yourself use Lombok. Lombok has annotations such as @Data which ensures getters and setters without creating it manually. It makes the code clean and more readable.

Reply:
Okay

Reply:
You did not mention which column represents which field. Also follow camel casing while naming a field in java. Field names should start with lowercases. For example `accountNumber`